be/src/olap/rowset/segment_v2/index_page.h

```diff
@@ -78,64 +78,80 @@ class IndexPageBuilder {
     // is the builder currently between finish() and reset()?
     bool _finished = false;
     faststring _buffer;
-    std::vector<uint32_t> _entry_offsets;
+    uint32_t _count = 0;
```

Comment:
should update comment to keep consistent.

Reply:
done